internal/rsl/rsl.go

```diff
@@ -362,6 +362,40 @@ func GetLatestNonGittufReferenceEntry(repo *git.Repository) (*ReferenceEntry, []
 	return targetEntry, annotations, nil
 }
 
+// GetLatestReferenceEntry returns the first reference entry
+func GetLatestReferenceEntry(repo *git.Repository) (*ReferenceEntry, []*AnnotationEntry, error) {
```

Comment:
Why is `GetLatestEntry` not enough?

Reply:
Since GetLatestEntry does not fetch the latest RSL reference entry, just any entry, reference or annotation.

Reply:
Does this only display reference entries?

Reply:
Currently the idea is that it only outputs reference entries, and added applicable annotations to the entry itself, kind of like this 

entry , hash

annotation(if exist)

entry, hash ...

Reply:
Doesn't GetReferenceEntriesInRange handle that case, where lastID is an annotation? Ideally, it should still return the ref entries in that range even if the end point isn't a ref entry. I think neither firstID nor lastID is required to be that of a reference entry FWIW.

Reply:
Yes it does, fixed